### ISSUE TITLE
Can we use this with chatGPT plus not with API?

### ISSUE THREAD

**Issue by FurkanGozukara** [yihong0618/bilingual_book_maker#187]
Can we use this with chatGPT plus not with API?

I want to translate subtitle files

vtt format or srt format

**Comment by yihong0618** [comment]
chatgpt plus not for now
srt will support later

**Comment by FurkanGozukara** [comment]
> chatgpt plus not for now srt will support later

chatgpt plus would be amazing. that way you have unlimited tokens

**Comment by yihong0618** [comment]
@FurkanGozukara FYI, if we use chatgpt plus maybe ban for openai 
that's why we still add it yet

see https://github.com/acheong08/ChatGPT/issues/1158

**Comment by FurkanGozukara** [comment]
> @FurkanGozukara FYI, if we use chatgpt plus maybe ban for openai that's why we still add it yet
> 
> see [acheong08/ChatGPT#1158](https://github.com/acheong08/ChatGPT/issues/1158)

They say that chatgpt plus is safe. Could you release a script that can translate given subtitle with the method you linked?

**Comment by yihong0618** [comment]
https://github.com/yihong0618/bilingual_book_maker/pull/43
you can use this pr”s repo

**Comment by pc418** [comment]
Yes, now you can: since #559, install the official Codex CLI, log in with your ChatGPT subscription, and translate without an API key:

```
python3 make_book.py --book_name book.srt --api_format codex --language zh-hans
```

One session is reused across the whole book and compacted automatically. Closing as shipped — please reopen if it doesn't cover your case.
